code/game/objects/structures/cryopod.dm

```diff
@@ -0,0 +1,86 @@
+/obj/structure/cryopod_spawner/proc/justequip(var/mob/living/carbon/human/H, var/title, var/alt_title, var/outfit_type)
+	var/decl/hierarchy/outfit/outfit = outfit_type
+	. = outfit.equip(H, title, alt_title)
+
+/obj/structure/cryopod_spawner
+	icon = 'icons/obj/machines/excelsior/objects.dmi'
+	icon_state = "cryopod"
+	var/mob/spawnmob
+	var/spawnname
```

Comment:
These var names are confusing since they have nothing to do with name/surname, it's faction/job

Reply:
Perhaps `var/spawn_role`?